src/Symfony/Component/ErrorHandler/ErrorHandler.php

```diff
@@ -175,7 +170,7 @@ public function __construct(BufferingLogger $bootstrappingLogger = null)
      * @param array|int       $levels  An array map of E_* to LogLevel::* or an integer bit field of E_* constants
      * @param bool            $replace Whether to replace or not any existing logger
      */
-    public function setDefaultLogger(LoggerInterface $logger, $levels = E_ALL, $replace = false)
+    public function setDefaultLogger(LoggerInterface $logger, $levels = E_ALL, bool $replace = false): void
```

Comment:
`int $levels`

Reply:
this method is called either with an int (E_* levels directly) or an array of those values, can't use int in this case. docBlock also states this behaviour.